src/index.scss

```diff
@@ -1,8 +1,29 @@
+@mixin inputfield {
```

Comment:
- [ ] Please, consider fixing the styling to be exactly the same as the original required design. You can see that in your version the navbar and books list section are not cover all page width - I mean you have the gray space around. Try to fix this issue - I know you can do it easily 👍 
- [ ] Please, also try to add a percentage complete view of each book, as it shows on the original design. 
- [ ] Additionally, there is should be a divider between the percentage completed section and the chapter info section - I marked this on the screenshot.
> You design
<img width="1427" alt="Screen Shot 2022-05-07 at 10 42 26" src="https://user-images.githubusercontent.com/49064106/167242534-4775aad2-04ac-46eb-9ead-311e3088d805.png">

>Original design
<img width="1085" alt="Screen Shot 2022-05-07 at 10 41 43" src="https://user-images.githubusercontent.com/49064106/167242539-71fb175f-1e9f-419a-a7bb-83bbdec78048.png">

Reply:
Hi @elmar8287. Thanks for the code review. Ihtought that part is also the part of the design since there is box-shadow behind. As you mentioned, I will implement that easily. Thank you again for the code review.